internal/wrapnix/wrapper.go

```diff
@@ -48,6 +48,7 @@ func CreateWrappers(ctx context.Context, args CreateWrappersArgs) error {
 	_ = os.MkdirAll(destPath, 0o755)
 
 	bashPath := cmdutil.GetPathOrDefault("bash", "/bin/bash")
+	sedPath := cmdutil.GetPathOrDefault("sed", "sed")
```

Comment:
Won't the default value trigger the bug again? Should the default be `/usr/bin/sed` to mirror `/bin/bash` above?

Reply:
Yeah if `sed` is not in path this will trigger same issue. Though if `sed` is not in path the wrapper would have failed anyway. `sed` is POSIX so I kinda assume it exists.

There's an additional problem that affects `sed` and `bash`. If you are using global and a project devbox and both install sed and/or bash, you could end up cycling between both wrappers recursivelly. For example:

* Add coreutils to project (bin wrappers use `/usr/bin/sed`)
* Add coreutils to global while shelled in to project (bin wrappers use `sed` in project)
* Make some change to project, now sed in bin wrapper points to global sed
* Cycle 

Another edge case:

* if 2 devbox processes are creating wrappers at the same time, theres a chance the second one will point to the bin wrapper incorrectly.

These are pretty edge cases, but would suck. I think a better solution is to just special case `sed` and `bash` and never bin wrap them. Thoughts?

Reply:
Follow up https://github.com/jetpack-io/devbox/pull/1571